packages/boxel-ui/public/assets/check-mark-teal.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><g stroke="rgba(0,0,0,0.1)" stroke-width="1"><path d="M12.727,6l-6,6L4,9.273" transform="translate(6 7) translate(-4 -6)" fill="none" stroke="var(--icon-color, #03c4bf)" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></g></svg>
```

Comment:
Could this be `check-mark` with a `--icon-color` override?

Reply:
I didn't know that was an option--I'll give it a shot